src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -87,13 +90,16 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                                 if(item == null){
                                     continue;
                                 }
-                                if(item.getType().toString().contains("ARROW")){
+                                if(isArrow(item)==true){
                                     item.setType(Material.TIPPED_ARROW);
                                     item.setAmount(item.getAmount());
-                                    PotionMeta potionmeta = (PotionMeta)item.getItemMeta();
-                                    potionmeta.setBasePotionData(new PotionData(PotionType.getByEffect(PotionEffectType.getByName(args[0]))));
-                                    potionmeta.addCustomEffect(new PotionEffect(PotionEffectType.getByName(args[0]),10,10),true);//duration,amplifire,Show particles
-                                    item.setItemMeta(potionmeta);
+                                    PotionMeta potionMeta = (PotionMeta)item.getItemMeta();
+                                    potionMeta.setBasePotionData(new PotionData(PotionType.getByEffect(PotionEffectType.getByName(args[0]))));
```

Comment:
args[0] is repeated a number of times and should be a variable

Reply:
done